### PR TITLE
Update PR description with site link instead of comment.

### PR DESCRIPTION
**Pull request by shrimpy**
Result will looks like below screenshot, we append a new line of text at the end of the description.

image

### CODE REVIEW

File: Slingshot.Api/Abstract/Repository.cs [hunk 1]
```diff
@@ -145,15 +145,15 @@ public virtual Task<bool> HasScmInfo()
...
-        public virtual Task WritePullRequestComment(string prId, string comment)
+        
+        public virtual Task<IPullRequestInfo> GetPullRequest(string prId)
         {
-            return Task.FromResult(new object());
+            return Task.FromResult<IPullRequestInfo>((IPullRequestInfo)new object());
         }
 
-        public virtual Task<T> GetPullRequest<T>(string prId) where T : IPullRequestInfo
+        public virtual Task UpdatePullRequest(string prId, dynamic content)
```
Comment (davidebbo) [inline Slingshot.Api/Abstract/Repository.cs]: Probably a bit cleaner to have it take an IPullRequestInfo. dynamic is an implementation detail of BitBucketPullRequest.

File: Slingshot.Api/Models/IPullRequestInfo.cs [hunk 2]
```diff
@@ -2,6 +2,10 @@
 {
     public interface IPullRequestInfo
     {
+        dynamic RawContent { get; }
```
Comment (davidebbo) [inline Slingshot.Api/Models/IPullRequestInfo.cs]: Should not be on the interface. It's not a meaningful contract.

File: Slingshot.Api/Abstract/Repository.cs (hunk 1 above)
Comment (shrimpy) [inline Slingshot.Api/Abstract/Repository.cs]: agree. fixed!

**Comment by davidebbo** [comment]
:shipit:

**Pull request merged**